hoot-core-test/src/test/cpp/hoot/core/TestUtils.cpp

```diff
@@ -66,7 +66,7 @@ bool TestUtils::compareMaps(const QString& refPath, const QString testPath)
   OsmXmlReader reader;
```

Comment:
There are a bunch of these where OsmXmlReader had a duplicated method that wasn't part of the reader interface, so removed it.